src/lib/strategies/FocusOnlyKeyEventStrategy.js

```diff
@@ -145,9 +145,9 @@ class FocusOnlyKeyEventStrategy extends AbstractKeyEventStrategy {
    * @param {HandlersMap} actionNameToHandlersMap - Map of actions to handler functions
    * @param {Object} options Hash of options that configure how the actions
    *        and handlers are associated and called.
-   * @returns {[FocusTreeId, ComponentId]} The current focus tree's ID and a unique
-   *         component ID to assign to the focused HotKeys component and passed back
-   *         when handling a key event
+   * @returns {FocusTreeId|undefined} The current focus tree's ID or undefined if the
```

Comment:
🐥 📚 Fixes stale documentation reporting the correct return type & adds a description of when the function will return `undefined`.